src/components/mission.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable react/prop-types */
+import { useDispatch } from 'react-redux';
+import { bookMission, cancelMission } from '../redux/missions/missions';
+
+const Mission = ({ mission }) => {
+  const dispatch = useDispatch();
+
+  const bookAction = (id) => {
+    dispatch(bookMission(id));
+  };
+
+  const cancelAction = (id) => {
+    dispatch(cancelMission(id));
+  };
+
+  return (
+    <tr key={mission.id} className="odd:bg-zinc-200">
+      <td className="border-2 border-zinc-400 p-2 font-bold text-[1.1rem]">{mission.mission_name}</td>
+      <td className="w-[60%] border-2 border-zinc-400 p-2">{mission.description}</td>
+      <td className="w-[15%] border-2 border-zinc-400 text-center">
+        <div className="flex justify-center">
+          {mission.reserved ? (
+            <p className="bg-sky-400 rounded-md p-2 text-white ">Active Member</p>
+          ) : (
+            <p className="text-white bg-zinc-500 p-2 rounded-md">NOT A MEMBER</p>
+          )}
+        </div>
```

Comment:
-  Please use React conditional rendering syntax for conditions instead of the Ternary operator as this is part of the requirements.
You can use the syntax:
`{ mission.reserved && ( // code for reserved rocket)}
{ !mission.reserved && ( // code for not reserved rocket)}`